server/src/pages/shot/shotpage-header.js

```diff
@@ -88,13 +88,15 @@ exports.ShotPageHeader = class ShotPageHeader extends React.Component {
   }
 
   renderFxASignIn() {
-    return (
-      this.props.isOwner ?
+    if (this.props.isOwner) {
```

Comment:
Are we choosing not to show the login button to Firefox users that aren't logged in, and are viewing someone else's page?

Reply:
That's correct, we are showing login button on owned shots for Firefox users, as documented in [design doc]([url](https://docs.google.com/document/d/1wxWNqHlQLUoeDrKqI5mUOSrhW-p2SN4_xwB2hRPk02k/edit#))